src/Directory.Build.props

```diff
@@ -13,10 +14,17 @@
   </PropertyGroup>
 
   <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>FakeItEasy.Analyzer</RootNamespace>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <FileAlignment>512</FileAlignment>

```

Comment:
AppDesignerFolder and FileAlignment are probably not necessary. They're no longer included in the default templates for SDK-style projects, I think they have sensible default values now.
Not really in scope for this PR, of course, so we can address this separately.